src/components/tabset/tabset.component.spec.ts

```diff
@@ -116,12 +121,56 @@ describe('Tabset Component', () => {
         expect(component.onTabDeselect).toHaveBeenCalled();
     });
 
+    it('should change to new tab when tab is programmatically selected by tab index', () => {
+
+        const tab1 = getTabForClass(0, nativeElement);
+        const tab3 = getTabForClass(2, nativeElement);
+
+        // check first tab is active
+        expect(tab1.classList.contains('active')).toBeTruthy();
+
+        spyOn(component, 'onTabActivated');
+
+        // programatically change tab by index
+        tabset.selectTab(2);
+
+        // check second tab is now active/first tab is not active
+        expect(tab3.classList.contains('active')).toBeTruthy();
+        expect(tab1.classList.contains('active')).toBeFalsy();
+
```

Comment:
I think the previous comments have been overlooked. Could you verify that the correct tab content (e.g. innertext of `h4`) is present?